endpoint.go

```diff
@@ -152,3 +168,16 @@ func (p Endpoint) File() (path string) {
 
 	return p.args.(string)
 }
+
+func (p Endpoint) TCP() (addr, host string, port int) {
+	if p.protocol != ProtocolTCP {
+		panic(Error{
+			Op:       "tcp",
+			Err:      ErrInvalidValue,
+			Protocol: p.protocol,
+			Values:   p.args,
+		})
+	}
+	hp := p.args.(hostPort)
+	return fmt.Sprintf("%s:%s:%d", p.protocol, hp.host, hp.port), hp.host, hp.port
```

Comment:
Maybe we need to use `<host>:<port>` as `addr` so that users can use them directly?

```suggestion
	return fmt.Sprintf("%s:%d", hp.host, hp.port), hp.host, hp.port
```

Reply:
done